### PR TITLE
Small corrections for ceph tests

### PR DESCRIPTION
**Pull request by legan4ik**

### CODE REVIEW

File: mk_verificator/tests/ceph/config.yaml [hunk 1]
```diff
@@ -1,2 +1,3 @@
 ---
-ceph_monitors: ['ceph-005*', 'ceph-006*', 'ceph-007*']
+ceph_monitors: ['ceph-003*', 'ceph-004*', 'ceph-005*']
```
Comment (msenin94) [inline mk_verificator/tests/ceph/config.yaml]: I suppose we can even move it to the global config, as we use information about ceph monitors not only here

File: mk_verificator/tests/ceph/test_ceph.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+from mk_verificator import utils
+
+
+def test_check_ceph_osd(local_salt_client):
+    config = utils.get_configuration(__file__)
+    osd_fail = \
```
Comment (msenin94) [inline mk_verificator/tests/ceph/test_ceph.py]: could you please share what salt returns? (you know, unfortunately I have no ceph under my hand =( )

File: mk_verificator/tests/ceph/config.yaml (hunk 1 above)
Comment (legan4ik) [inline mk_verificator/tests/ceph/config.yaml]: Where?

File: mk_verificator/tests/ceph/test_ceph.py (hunk 2 above)
Comment (legan4ik) [inline mk_verificator/tests/ceph/test_ceph.py]: e.g. output of the test
E       AssertionError: Some osds are in down state
E       assert not 'XX  0.XXXX         osd.XX       down        0          1.00000'

**Pull request merged**